### PR TITLE
Module1: Hello Microverse!

### PR DESCRIPTION
**Pull request by hammaazarok**
* added Linter
* added gitignore 
* created the home page
* added markdown template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,35 @@
...
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
```
Comment (zhadier) [inline README.md]: - [x] Kindly keep the Getting started and Prerequisites section in your ReadMe, all headers that can be present should be present inside the ReadMe. You can add steps to clone the repo and run the live server in the Getting started section :)

You can use something like this, kindly personalize it for your Project


## Getting Started
#### Step 1:

- Clone this repo locally using git in the command line and the following command

  `git clone https://github.com/zhadier/My-Portofolio`
  
- Alternatively, you can just download the complete zip file and extract the folder in your directory

#### Step 2:
Copy All Project files and run using VSCODE. HTML can be viewed in browser.


### Prerequisites
- VSCODE

File: README.md [hunk 2]
```diff
@@ -1,72 +1,35 @@
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
```
Comment (zhadier) [inline README.md]: - [x] This issues link would work automatically, you should keep as much content in the ReadMe as you can from the template. Kindly add this in the ReadMe as well.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+test.md
+app_screenshot.png
```
Comment (zhadier) [inline .gitignore]: - [x] [_Optional_] Add `node_modules/` aswell in the .gitignore so they never get pushed to the repo, even by mistake.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @hammaazarok  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ No linter errors
✔️ GitHub flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (hammaazarok) [reply README.md]: Done, Thank you @zhadier .

File: README.md (hunk 2 above)
Comment (hammaazarok) [reply README.md]: Done, Thank you @zhadier .

File: .gitignore (hunk 3 above)
Comment (hammaazarok) [reply .gitignore]: Done, Thank you @zhadier .

File: README.md [hunk 4]
```diff
@@ -1,56 +1,37 @@
...
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+step 1:
+Clone this repo locally using git in the command line and the following command
 
+git clone https://github.com/hammaazarok/hello-microverse.git
 
-To get a local copy up and running follow these simple example steps.
+Alternatively, you can just download the complete zip file and extract the folder in your directory
```
Comment (mukhammadrizooff) [inline README.md:14]: Good job @hammaazarok ! You did very well in the requirements mentioned by the previous code reviewer 👍

**Review by mukhammadrizooff — APPROVED** [review]
Hi @hammaazarok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**